talk/owt/sdk/include/cpp/owt/base/logging.h

```diff
@@ -35,6 +35,13 @@ class OWT_EXPORT Logging final {
   static void LogToConsole(LoggingSeverity severity);
   /// Set logging to files under provided dir rotately.
   static void LogToFileRotate(LoggingSeverity severity, std::string& dir, size_t max_log_size);
+  /// Set logging to files under provided dir rotately, with file name starting
+  /// with the 'prefix`.
+  static void LogToFileRotate(LoggingSeverity severity,
+                              std::string& dir,
```

Comment:
`dir` and `prefix` could be `const string&`, although the previous API is not const. (it should be const as well)

Reply:
done. Since CG depend on this, I didn't modify the method signature of the other version of API.